client/components/drop-zone/index.jsx

```diff
@@ -15,6 +15,9 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import RootChild from 'components/root-child';
+import {
+	showDropZone, hideDropZone
+} from 'state/components/drop-zone/actions';
```

Comment:
Totally a nitpick, but either keep it the statement in one line or have one action per line.